src/features/page_view_timing/aggregate/index.js

```diff
@@ -180,8 +102,8 @@ export class Aggregate extends AggregateBase {
     Mitigation: We've set initial CLS to null so that it's omitted from timings like 'pageHide' in that edge case. It should only be included if onCLS callback was executed at least once.
     Future: onCLS value changes should be reported directly & CLS separated into its own timing node so it's not beholden to 'pageHide' firing. It'd also be possible to report the real final CLS.
     */
-    if (this.cls !== null) {
-      attrs.cls = this.cls
+    if (cumulativeLayoutShift.value.current >= 0) {
```

Comment:
The default `value.current` in which CLS has never been set yet is now `undefined` via `#value[-1]` of an empty array. This should check `!== undefined` then rather than 0.